src/cordova/android/README.md

```diff
@@ -6,17 +6,19 @@ The main entrypoint to Android's Java code is `cordova-plugin-outline/android/ja
 
 ## Building the Android app
 
+> 💡 NOTE: To best get set up for Android, use `tools/build/setup_macos_android.sh` or `tools/build/setup_linux_android.sh`.
+
 Additional requirements for Android:
 
+- [Java JDK 11](https://jdk.java.net/archive/)
+- Latest [Android Sdk Commandline Tools](https://developer.android.com/studio/command-line) ([download](https://developer.android.com/studio#command-line-tools-only))
+  - Place it at `$HOME/Android/sdk/cmdline-tools/latest`
```

Comment:
*minor* On my machine the SDK folder name is capitalized (I think that's the default name created by Android Studio):

```suggestion
  - Place it at `$HOME/Android/Sdk/cmdline-tools/latest`
```